Src/AutoNSubstituteUnitTest/AutoConfiguredFixtureIntegrationTest.cs

```diff
@@ -427,23 +427,5 @@ public void InterfaceShadowedMethodsReturnValueFromFixture()
             Assert.Same(frozenString, (result as IInterfaceWithShadowedMethod).Method(0));
             // Teardown
         }
-
-        [Fact]
-        public void InterfacesImplementingIEnumerableReturnFiniteSequence()
```

Comment:
Why delete this test? Doesn't it represent a feature that we want to ensure keeps working?